Features/Standard/SubitemGrid.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Subitem Grid
+
+    @javascript
+    Scenario: The Subitem Grid view is used when viewing a Content item which Content Type belongs to the Media Content Type Group
+        Given a Content Type of the Media Group flagged as container
+          And a Content item of this type
+          And this Content item has a subitem
+         When I view this Content
+         Then The Subitem Grid is visible
```

Comment:
I don't think that "visible" is a good choice there. To me, it implies that it is otherwise "hidden". It's not really hidden/visible, more of a choice out of a list.

"The the Subitems are shown using the Grid view"